### ISSUE TITLE
The email domain is not in whitelist [error-invalid-domain]"

### ISSUE THREAD

**Issue by asarubbo** [RocketChat/Rocket.Chat.Livechat#649]
Hello,

the livechat plugin does not work anymore since it does not accept domains outside the whitelist. I never enabled whitelist/blacklist for livechat.

curl output when we try tojoin livechat:
```
{
  "isClientSafe": true,
  "error": "error-invalid-domain",
  "reason": "The email domain is not in whitelist",
  "details": {
    "function": "RocketChat.validateEmailDomain"
  },
  "message": "The email domain is not in whitelist [error-invalid-domain]",
  "errorType": "Meteor.Error",
  "success": false
}
```

Screenshot of the settings:
![Screenshot_20210923_102256](https://user-images.githubusercontent.com/4741819/134475864-9d6c16ec-a821-4b94-841e-ab210626bf98.png)

**Comment by dudanogueira** [comment]
Hi!

Can you please put the curl command you used?

Also, are you at the latest (3.18.1) version?

**Comment by asarubbo** [comment]
Hello, the curl command is nothing more than what comes from the chrome inspector (Copy as Curl), anyway this is something like:

```
curl 'https://$HOST/api/v1/livechat/visitor' \
  -H 'authority: $HOST' \
  -H 'sec-ch-ua: "Google Chrome";v="93", " Not;A Brand";v="99", "Chromium";v="93"' \
  -H 'dnt: 1' \
  -H 'sec-ch-ua-mobile: ?0' \
  -H 'user-agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/93.0.4577.82 Safari/537.36' \
  -H 'sec-ch-ua-platform: "Linux"' \
  -H 'content-type: application/json' \
  -H 'accept: */*' \
  -H 'origin: $HOST' \
  -H 'sec-fetch-site: same-origin' \
  -H 'sec-fetch-mode: cors' \
  -H 'sec-fetch-dest: empty' \
  -H 'referer: $HOST/livechat' \
  -H 'accept-language: it,en-US;q=0.9,en;q=0.8,de;q=0.7,pl;q=0.6,fr;q=0.5,es;q=0.4,mt;q=0.3,tr;q=0.2' \
  --data-raw '{"visitor":{"name":"agostino","email":"agostino@agostino.it","department":"wQNRdv36NGK3BHzof","token":"us5vif6k7asdaew3qf23432412"}}' \
  --compressed

```

I'm on 3.18.0 and I will update asap.

**Comment by asarubbo** [comment]
this is fixed in 3.18.1
